config/config.php

```diff
@@ -4,25 +4,25 @@
 $this['base_path'] = dirname(__DIR__).'/';
 
 /** Import core settings */
-$this->addFile(__DIR__ . '/config.json');
-
-/** Import middlewares */
-$this->addFile(__DIR__ . '/middlewares.json');
-
-/** Import routes */
-$this->addFile(__DIR__ . '/routes.json');
-
-/** Import redirections */
-$this->addFile(__DIR__ . '/redirections.json');
+$this->addFile(__DIR__ . '/common/config.json');
 
 /** Import templates */
-$this->addFile(__DIR__ . '/templates.json');
+$this->addFile(__DIR__ . '/common/templates.json');
 
 /** Import attachments */
-$this->addFile(__DIR__ . '/attachments.json');
+$this->addFile(__DIR__ . '/common/attachments.json');
 
 /** Import local settings */
 $appEnv = 'local';
-if (file_exists(__DIR__.'/config.'.$appEnv.'.json')) {
-    $this->addFile(__DIR__.'/config.'.$appEnv.'.json');
+if (file_exists(__DIR__ . '/common/config.'.$appEnv.'.json')) {
+    $this->addFile(__DIR__ . '/common/config.'.$appEnv.'.json');
+}
+
+$this['adminPath'] ??= 'admin';
+$isAdmin = $_SERVER['REQUEST_URI'] == '/'.$this['adminPath'] || strstr($_SERVER['REQUEST_URI'], '/'.$this['adminPath'].'/') !== false;
```

Comment:
This condition `$_SERVER['REQUEST_URI'] == '/'.$this['adminPath']` risks matching a content page (such as `/administration`).

Reply:
```suggestion
$isAdmin = preg_match('!^/'.preg_quote($this['adminPath'], '\b!').'!', $_SERVER['REQUEST_URI']);
```